recipes/requests-gracedb/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "requests-gracedb" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 76dc4fa35bb150e8c31bb21a4c5577ed3336a09067bc6fa0371973aff5f2e22f
+  patches:
+    # https://git.ligo.org/emfollow/requests-gracedb/issues/13
+    - namespace_packages.patch
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools >=38.3.0
+  run:
+    - cryptography
+    - python
+    - requests
+    - safe-netrc
+    - six
+
+test:
+  imports:
+    - requests_gracedb
+  requires:
+    - pytest
+    - pytest-freezegun
+    - pytest-httpserver
+    - pytest-socket
+    - python >=3
```

Comment:
I guess the package supports Python 2 and 3 but the tests only support Python 3?

(It's not a problem, just really unusual)

Reply:
No, the tests support python2, but need `mock` for that, and I thought that this was better (read: easier) than adding `- mock  # [py<30]`. Happy to change if requested.

Reply:
Ah that makes sense! LGTM 👍